resources/public/api/conf/5.0/foreign_property_cumulative_summary_create_and_amend.yaml

```diff
@@ -37,17 +36,14 @@ requestBody:
   content:
     application/json:
       schema:
-        $ref: './schemas/foreign_property_create_or_amend_cumulative/def1/request.json'
+        $ref: './schemas/foreign_property_cumulative_summary_create_and_amend/def1/request.json'
       examples:
-        full-expenses-request:
-          description: Full Expenses request for TY 2025-26
+        Full expenses request for TY 2025-26 and after:
           value:
-            $ref: './examples/foreign_property_create_or_amend_cumulative/def1/request.json'
-        consolidated-expenses-request:
-          description: Consolidated Expenses request for TY 2025-26
+            $ref: './examples/foreign_property_cumulative_summary_create_and_amend/def1/request.json'
+        Consolidated expenses request for TY 2025-26 and after:
           value:
-            $ref: './examples/foreign_property_create_or_amend_cumulative/def1/consolidated_request.json'
-
+            $ref: './examples/foreign_property_cumulative_summary_create_and_amend/def1/consolidated_request.json'
 
 responses:
   "201":
```

Comment:
Not related to this but this is meant to be a 204 that we returned so can this be updated as part of this PR? Thanks